aptos-move/aptos-vm/src/aptos_vm_impl.rs

```diff
@@ -250,6 +230,7 @@ impl AptosVMImpl {
                             .any(|(_, hash)| hash == txn_data.script_hash)
                     })
                     .unwrap_or(false);
+                const MAXIMUM_APPROVED_TRANSACTION_SIZE: u64 = 1024 * 1024;
```

Comment:
I don't think constants should be declared inline? was there a specific reason to move it here?

Reply:
Not really, moved back👍